esp/esp/program/modules/handlers/commmodule.py

```diff
@@ -175,7 +181,10 @@ def commfinal(self, request, tl, one, two, module, extra, prog):
                                                       sendto_fn_name  = sendto_fn_name,
                                                       sender     = fromemail,
                                                       creator    = request.user,
-                                                      msgtext    = body,
+                                                      msgtext = unicode(loader.get_template('email/default_email_html.txt').render(DjangoContext(
+                                                                                                                                       {'msgbdy': body,
+                                                                                                                                        'user': request.user,
```

Comment:
This template should get rendered twice.  First, here, we render `msgbdy` into `default_email_html.txt` to get `msgtext`.  This happens once, and therefore should not include any user data, because we don't yet know what user the email will be sent to.  Then, later, below, we the output of that (`msgtext`) into the actual email, once for each user, injecting their information into the template.  Any variables in `msgtext` will get interpreted on the second pass.  When you include `request.user` the first time, it means that any user variables in `default_email_html.txt` get rendered according to the user *sending* the email, because that's who `request.user` is; instead you want to render them the second time with the email's recipient as the user.

I wrote too quickly above and was a bit incorrect: any template variables that you put in `default_email_html.txt` that should get rendered on the *second* pass, i.e., any user variables, need to have their template tags excaped, using `{% templatetag openvariable %}` instead of `{{` and `{% templatetag closevariable %}` instead of `}}`.  This way, the first rendering pass will convert those to `{{` and `}}` and then the second rendering pass will see them as normal variables, and interpolate the correct user's information.